### PR TITLE
bump up @storybook/components and @storybook/manager-api

### PR DESCRIPTION
**Pull request by trfv**

### CODE REVIEW

**Comment by trfv** [comment]
@kemuridama 
お手隙で確認お願いします 🙏

File: src/GitHub.tsx [hunk 1]
```diff
@@ -26,8 +26,10 @@ export const GitHub = ({ active, api }: Props) => {
   ].join("/");
 
   return (
-    <IconButton active={active} href={url} title="GitHub" target="_blank">
-      <Icons icon="github" />
+    <IconButton active={active}>
+      <a href={url} title="GitHub" target="_blank" rel="noreferrer">
+        <Icons icon="github" />
+      </a>
```
Comment (trfv) [inline src/GitHub.tsx:32]: cf. https://github.com/storybookjs/storybook/blob/next/code/ui/components/src/components/IconButton/IconButton.stories.tsx#L70-L81

Comment (trfv) [inline src/GitHub.tsx:32]: cf. https://github.com/storybookjs/storybook/pull/24266

**Review by kemuridama — APPROVED** [review]
lgtm

**Pull request merged**